### ISSUE TITLE
Screen goes blank when trying to play

### ISSUE THREAD

**Issue by Hiryuto** [processing/p5.js-web-editor#2296]
### p5.js version

v1.7.0

### What is your operating system?

Mac OS

### Web browser and version

114.0.5735.198 (Official Build) （x86_64） 

### Actual Behavior

When you enter the code and press the play button, 
everything disappears, including the editor, and
you get a blank screen.
This phenomenon did not occur before v2.7.0 of the Web Editor, 
so we believe it is caused by the changes made in v2.7.0.

### Expected Behavior

![20230715-0241_00a66095ab1993e540f67a40ab154501](https://github.com/processing/p5.js-web-editor/assets/108036759/197a170f-7eef-4414-b6cc-976572aeaf76)

### Steps to reproduce

The frequency with which the problem occurs is unknown, as is how to reproduce it.
It seems to happen most often when repeating "edit code and play".

**Comment by Hiryuto** [comment]
When I play it, all the child elements of `<div id="root" class="root-app">` disappear, so I assume they have turned white

**Comment by jht9629-nyu** [comment]
Get blank screen with script error in p5js editor v2.7.0
Sample sketch to repo the failure:
https://editor.p5js.org/jht9629-nyu/sketches/4DETmJP-l
```
function setup() {
  createCanvas(400, 400);
  foo() // error
}

function draw() {
  background(220);
}
```

**Comment by jht9629-nyu** [comment]
I see this in the console log. 
Appears that Editor code is not able to deal with failure in this expression

            const errorObj = { stack: consoleEvent.data[0].toString() };

TypeError: Cannot read properties of undefined (reading 'toString')
    at Editor.jsx:273:60
    at Array.forEach (<anonymous>)
    at o.value (Editor.jsx:269:34)
    at ju (react-dom.production.min.js:251:463)
    at t.unstable_runWithPriority (scheduler.production.min.js:18:437)
    at Yi (react-dom.production.min.js:120:325)
    at Bu (react-dom.production.min.js:244:437)
    at Eu (react-dom.production.min.js:223:420)
    at react-dom.production.min.js:121:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:437)
vs @ react-dom.production.min.js:196
react

**Comment by matdas** [comment]
Yeah, i am getting the same issue.

**Comment by 0000marcosg** [comment]
Im getting the same error.
For me the issue appear when there is a error in the code. Instead of show the error in the console, the whole editor chrashes. 

Tested in Chrome 114.0.5735.199 and Microsoft Edge 114.0.1823.82

**Comment by raclim** [comment]
Thank you so much for reporting this, I'm really sorry that this issue happened due to an oversight in upgrading the console-feed package 😭 (further details about the issue and its fix in https://github.com/processing/p5.js-web-editor/issues/2298)

I just deployed a new release which seems to resolve this issue for now, but please reopen this issue if it persists!

**Comment by jht9629-nyu** [comment]
Thanks for the quick fix!Checks out ok so far.-{john-henry}-[thompson]On Jul 16, 2023, at 12:04 AM, raclim ***@***.***> wrote:﻿
Thank you so much for reporting this, I'm really sorry that this issue happened due to an oversight in upgrading the console-feed package 😭 (further details about the issue and its fix in #2298)
I just deployed a new release which seems to resolve this issue for now, but please reopen this issue if it persists!
